<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="toggle_check_exit">
            <property name="text">
             <string>Confirm exit while emulation is running</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="updateBox">
       <property name="title">
        <string>Updates</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_update">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_update">
          <item>
           <widget class="QCheckBox" name="toggle_update_check">
            <property name="text">
             <string>Check for updates on start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_auto_update">
            <property name="text">
             <string>Silently auto update after closing</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Emulation</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Region:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="region_combobox">
              <item>
               <property name="text">
                <string>Auto-select</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">JPN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">USA</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">EUR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">AUS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">CHN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">KOR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">TWN</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
         <item>
           <widget class="QCheckBox" name="toggle_use_priority_boost">
             <property name="text">
               <string>Boost proirity on starved threads</string>
             </property>
           </widget>
         </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Hotkeys</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="GHotkeysDialog" name="hotkeysDialog" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="button_reset_defaults">
       <property name="text">
        <string>Reset All Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GHotkeysDialog</class>
   <extends>QWidget</extends>
   <header>hotkeys.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
